docker/notifications/crontab.txt

```diff
@@ -0,0 +1 @@
+*/2 * * * * /etc/trustpoint/notifications/execute_notifications.sh >> /var/log/cron.log 2>&1
```

Comment:
Wouldn't it be better to run this as `www-data` instead of root? I tried several times to change the user (e.g. by adding ` www-data ` between `*/2 * * * * ` and the command, but it does not work (`/bin/sh: 1: www-data: not found`)